app/views/monsters/show.html.erb

```diff
@@ -30,6 +30,14 @@
   <ul>
     <% @reviews.each do |review| %>
     <li><%= render partial: 'review', locals: {review: review} %></li>
+    <div>
+      <%= render partial: 'upvote', locals: {review: review} %> <%= render partial: 'downvote', locals: {review: review} %>
+    </div>
+    <div>
+      <% upvotes = Vote.where(review_id: review.id, value: "+") %>
```

Comment:
I would try to do this in the controller. You can assign the instance variable `@upvotes` and then use it directly on line 39. Same for `downvotes`